src/anonymousUserTracking/criteriaCompletionChecker.ts

```diff
@@ -362,7 +362,7 @@ class CriteriaCompletionChecker {
               return this.evaluateComparison(
                 query.comparatorType,
                 valueFromObj,
-                query.value ? query.value : ''
+                query.value ? query.values : ''
```

Comment:
why is this one slightly different than the others?

Reply:
Because in value we have single element and in values we have multiple element(s),
and when we create criteria for the isOneOf and isNotOneOf comparator we get values instead of value